Main() {
    //try {
        try {
            arr := bool.array(1)({true});
            return arr[1];
            //throw Standard.Error.ArrayIndexOutOfBounds();
        } catch(fail : Standard.Error.ArrayIndexOutOfBounds.lvalue) {
            return true;
        }
    //} catch(...) {
    //    return false;
    //}
}